source/systems/townclaimmodes.md

```diff
@@ -87,13 +87,13 @@ Let's begin by explaining the basics of this system:
 
 **5-** Placement of new huts can only be within the existing claims. As long as the Hut block is inside the existing claim, it will place successfully (even if the building itself is partially outside of the claim when viewing with building tool).
 
-**6-** Breaking the TownHall block and placing it elsewhere will also remove it's claim and any building outside of it's new claim will not work.
+**6-** Breaking the TownHall block and placing it elsewhere will also remove its claim and any building outside of its new claim will not work.
 
-**7-** If for some reason two players are too close and the claim seems to "overlap", the player that placed his claim first will keep the claim, the other player will simply **not** expand into an existing claim. Unless the existing claim removes the claim (by removing a building), then the new "overlapping claim will take effect as ther ewould be **no** claim in that space.
+**7-** If for some reason two players are too close and the claim seems to "overlap", the player that placed his claim first will keep the claim, the other player will simply **not** expand into an existing claim. Unless the existing claim removes the claim (by removing a building), then the new "overlapping" claim will take effect as there would be **no** claim in that space.
 
-**Note:** An interesting feature is that in dynamic claim mode, when placing the Townhall there is NO warning of being too close to another Town. Townhalls CAN be close to each other (outside each other's claim of course) thus allowing very close Town collaborations.
+**Note:** An interesting feature is that in dynamic claim mode, when placing the Townhall, there is NO warning of being too close to another Town. Townhalls CAN be close to each other (outside each other's claim of course) thus allowing very close Town collaborations.
 
-Since there is NO warning of "too close to another Town" to place TownHall tho (like in the static claim mode) it is up to the players to always check their surrounding areas for existing Towns and for players that might not be paying attention and trying to establish a Town or getting too close to an existing Town.
+Since there is NO warning of "too close to another Town" to place Townhall tho (like in the static claim mode) it is up to the players to always check their surrounding areas for existing Towns and for players that might not be paying attention and trying to establish a Town or getting too close to an existing Town.
```

Comment:
```suggestion
Since there is NO warning of "too close to another Town" to place Town Hall though (like in the static claim mode) it is up to the players to always check their surrounding areas for existing Towns and for players that might not be paying attention and trying to establish a Town or getting too close to an existing Town.
```